### PR TITLE
Add example code for REP

### PR DESCRIPTION
**Pull request by DesmonDay**
Provide official code for paper **Simple and Effective Relation-based Embedding Propagation for Knowledge Representation Learning**.

### CODE REVIEW

File: examples/REP/ote_orth.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import paddle
+
+
+class Orth_OTE():
```
Comment (Yelrose) [inline examples/REP/ote_orth.py:19]: OrthOTE

**Review by kirayummy — CHANGES_REQUESTED** [review]
place the REP to /apps/Graph4KG/

**Review by Yelrose — APPROVED** [review]
LGTM

File: apps/Graph4KG/examples/REP/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+<h2 align="center">
+<img align="center"  src="./rep_method.png" alt="rep" width = "600" height = "350">
+</h2>
+
+## How to get knowledge embeddings
+
+Before running REP, we need to get pre-trained triplet-based embeddings. 
+
+In our paper, for FB15k-237 and WN18RR datasets, we use [KGEmbedding-OTE](https://github.com/JD-AI-Research-Silicon-Valley/KGEmbedding-OTE) to train KG embeddings, including 5 models we use in paper. For the best model config for TransE, RotatE and DistMult, we can find it [here](https://github.com/DeepGraphLearning/KnowledgeGraphEmbedding/blob/master/best_config.sh). For OTE and GC-OTE, we can find it [here](https://github.com/JD-AI-Research-Silicon-Valley/KGEmbedding-OTE).
```
[the remark this answers is not in the document]
Comment (DesmonDay) [reply apps/Graph4KG/examples/REP/README.md]: done

**Review by kirayummy — APPROVED** [review]

**Pull request merged**